### PR TITLE
restore changes in document contents

### PR DESCRIPTION
**Pull request by tvpeter**
- Restore changes in the body of the document made while fixing metadata information
- Preserve metadata changes made in #205 
- This closes #205

### CODE REVIEW

File: scalingbitcoin/tokyo-2018/atomic-swaps.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 title: Atomic Swaps
 transcript_by: Bryan Bishop
 categories: ['conference']
+tags: ['atomic swaps']
+speakers: ['Thomas Eizinger', 'Lloyd Fournier', 'Phillip Hoenisch']
```
Comment (kouloumos) [inline scalingbitcoin/tokyo-2018/atomic-swaps.md:6]: it seems that only the first one is the actual speaker

File: scalingbitcoin/tokyo-2018/edgedevplusplus/bulletproofs.md [hunk 2]
```diff
@@ -2,11 +2,10 @@
 title: Bulletproofs
 transcript_by: Bryan Bishop
 categories: ['conference']
+speakers: ['Karl-Johan Alm']
```
Comment (kouloumos) [inline scalingbitcoin/tokyo-2018/edgedevplusplus/bulletproofs.md:5]:
```suggestion
speakers: ['Kalle Alm']
```

File: scalingbitcoin/tokyo-2018/edgedevplusplus/coin-selection.md [hunk 3]
```diff
@@ -2,12 +2,9 @@
 title: Coin Selection
 transcript_by: Bryan Bishop
 categories: ['conference']
+speakers: ['Karl-Johan Alm']
```
Comment (kouloumos) [inline scalingbitcoin/tokyo-2018/edgedevplusplus/coin-selection.md:5]:
```suggestion
speakers: ['Kalle Alm']
```

File: scalingbitcoin/tokyo-2018/edgedevplusplus/lightning-network.md [hunk 4]
```diff
@@ -3,12 +3,11 @@ title: Lightning Network
 transcript_by: Bryan Bishop
 categories: ['conference']
 tags: ['lightning']
+speakers: ['Thaddeus Dryja']
```
Comment (kouloumos) [inline scalingbitcoin/tokyo-2018/edgedevplusplus/lightning-network.md:6]:
```suggestion
speakers: ['Tadge Dryja']
```

**Review by kouloumos — COMMENTED** [review]
Looks good! I appreciate the extra effort to modify a few of the titles in order to be more descriptive. 

I have a few comments,  and after that I believe we are ready!🚀

**Review by adamjonas — COMMENTED** [review]
I did not check every file but this is directionally much closer to what I had in mind.

Good job.

File: scalingbitcoin/tokyo-2018/atomic-swaps.md [hunk 5]
```diff
@@ -3,7 +3,7 @@ title: Atomic Swaps
 transcript_by: Bryan Bishop
 categories: ['conference']
 tags: ['atomic swaps']
-speakers: ['Thomas Eizinger', 'Lloyd Fournier', 'Phillip Hoenisch']
```
Comment (adamjonas) [inline scalingbitcoin/tokyo-2018/atomic-swaps.md]: why were these other speakers removed?

**Review by adamjonas — COMMENTED** [review]
ACK

Did a brief scan. I think it's good enough and we should squash and merge.

File: london-bitcoin-devs/2020-04-22-socratic-seminar.md [hunk 6]
```diff
@@ -3,7 +3,6 @@ title: Socratic Seminar - Grokking Bitcoin
 transcript_by: Michael Folkson
 categories: ['meetup']
 date: 2020-04-22
-speakers: ['Kalle Rosenbaum']
```
[the remark this answers is not in the document]
Comment (tvpeter) [reply london-bitcoin-devs/2020-04-22-socratic-seminar.md]: @kouloumos said he is not sure he was the speaker https://github.com/bitcointranscripts/bitcointranscripts/pull/211#discussion_r1026590481

File: scalingbitcoin/tokyo-2018/atomic-swaps.md (hunk 5 above)
Comment (kouloumos) [reply scalingbitcoin/tokyo-2018/atomic-swaps.md]: It seems that I've skipped verification on this. I was wrong
Screenshot 2022-11-18 at 10 42 35 PM
[src](https://tokyo2018.scalingbitcoin.org/)

File: london-bitcoin-devs/2020-04-22-socratic-seminar.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (kouloumos) [reply london-bitcoin-devs/2020-04-22-socratic-seminar.md]: Also wrong on this one. [src](https://www.meetup.com/BitDevsLDN/events/267668704/)

**Pull request merged**